### DISCUSSION TITLE
Could a panel tooltip also activate the tooltip of a different panel?

### DISCUSSION THREAD

**Discussion by sebasmagri** [grafana/grafana#48497] [Dashboards, Panels & Visualizations]
I'd like to have the tooltip of a different panel, or even all panels in a dashboard, activated when any tooltip is activated in any panel. Is this currently possible in any way? Would this be a feasible/attractive feature?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
